unittests/data/type_traits.hpp

```diff
@@ -544,13 +586,14 @@ namespace yes{
     struct x{
         x(){}
     };
+#ifdef __castxml__
+    typedef details::const_item const_item;
+    typedef details::const_container const_container;
+#endif
```

Comment:
Brad fixed this for C++03 mode also, and I can confirm that using the latest CastXML, I can move the two declarations above to `namespace no` and the test passes.

Also, [these lines](https://github.com/gccxml/pygccxml/blob/develop/unittests/type_traits_tester.py#L360-L365) in *type_traits_tester.py* need to be modified since `len(ci.declarations) == 3` for both GCCXML and CastXML now.

So how do you want to handle this fix?